src/index.ts

```diff
@@ -375,35 +375,33 @@ export const getTextFromHtml = (htmlString: string): string => {
 };
 
 /**
- * Formats a timestamp into a string representation of date and time in the format: DD-MM-YYYY HH:mm.
- * The timestamp is adjusted to the Indian Standard Time (IST) timezone.
+ * Formats a timestamp into a string representation of date and time according to region in the format: DD-MM-YYYY HH:mm.
+ * The timestamp is adjusted to the timezone of that region.
```

Comment:
We should not mention words like region or timezone at all. It is only creating confusion. The function is simply convert 1 representation into another. @divyjainjd

Reply:
Okay  @jainarpit014

Reply:
Removed this @jainarpit014